apps/nextjs-website/stories/MediaResources/default.stories.tsx

```diff
@@ -0,0 +1,198 @@
+import { Meta, StoryFn } from '@storybook/react';
+import MediaResources from '@react-components/components/MediaResources/MediaResources';
+import { MediaResourcesProps } from '@react-components/types/MediaResources/MediaResources.types';
+
+const meta: Meta<typeof MediaResources> = {
+  title: 'Components/MediaResources/Default',
+  component: MediaResources,
+};
+export default meta;
+
+const Template: StoryFn<MediaResourcesProps> = (args) => (
+  <MediaResources {...args} />
+);
+
+export const MediaResourcesFull = Template.bind({});
```

Comment:
```suggestion
export const MediaResourcesFull: StoryFn<typeof MediaResources> = Template.bind({});
```
Missing this type annotation will break NextJS and static website builds.